5.drug-dependency/scripts/4.drug_latent_dim_correlation.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import joblib
+import pandas as pd
+from scipy.stats import pearsonr
+from sklearn.preprocessing import MinMaxScaler
+import pathlib
+import sys
+import torch
+from torch.utils.data import DataLoader, TensorDataset
+import numpy as np 
+
+script_directory = pathlib.Path("../2.train-VAE/utils/").resolve()
+sys.path.insert(0, str(script_directory))
+from betavae import  extract_latent_dimensions
+from betatcvae import tc_extract_latent_dimensions
+from vanillavae import vvae_extract_latent_dimensions
+
+from utils import load_utils
+
+sys.path.insert(0, "../utils/")
+from data_loader import load_model_data
```

Comment:
This imports section could use some cleaning up. Consider using `isort` and adding a pre-commit file to do this automatically :D. 

If not please reference PEP8 styling guide for python code best practices! https://peps.python.org/pep-0008/#imports